test/controllers/static_pages_controller_test.rb

```diff
@@ -0,0 +1,13 @@
+require "test_helper"
+
+class StaticPagesControllerTest < ActionDispatch::IntegrationTest
```

Comment:
test don't need to do
delete it please

Reply:
sorry but i dont know which one to delete ?
![image](https://user-images.githubusercontent.com/87749692/130419698-a30de94f-22cf-461d-8771-38deeefd1b63.png)

Reply:
you should delete folder ../test/controllers